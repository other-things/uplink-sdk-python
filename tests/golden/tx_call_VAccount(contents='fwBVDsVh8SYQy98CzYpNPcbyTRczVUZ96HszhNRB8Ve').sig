r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 18425268736938265429589588007069347279121249928370974861001789595325158453682
