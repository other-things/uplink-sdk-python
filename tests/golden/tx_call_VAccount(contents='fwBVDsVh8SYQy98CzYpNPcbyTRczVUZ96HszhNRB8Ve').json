{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
                "args": [
                    {
                        "contents": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
                        "tag": "VAccount"
                    }
                ],
                "method": "get"
            },
            "tag": "Call"
        },
        "tag": "TxContract"
    },
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE0xODQyNTI2ODczNjkzODI2NTQyOTU4OTU4ODAwNzA2OTM0NzI3OTEyMTI0OTkyODM3MDk3NDg2MTAwMTc4OTU5NTMyNTE1ODQ1MzY4Mg=="
}